README.md

```diff
@@ -1,5 +1,41 @@
-# k8s-tensorflow-app
+# k8s-inception-ansible
 
-This ansible role deploys a tensorflow notebook which accesses tensorflow serving based inception architecture to classify images of cats from http://thecatapi.com. The script to deploy tensorflow serving containers are derived from this [repository](https://github.com/markgoddard/magnum-tools/tree/master/k8s-demo).
+This ansible role deploys a jupyter notebook that interfaces with a tensorflow
+server using a tensorflow client to classify images of cats from
+http://thecatapi.com.
 
-After the notebook is deployed, a hyperlink to the notebook will be shown on the Ansible logs as the final task.
+
+The script to deploy tensorflow serving containers are derived from this
+[repository](https://github.com/markgoddard/magnum-tools/tree/master/k8s-demo).
+
+This role was originally conceived as a component of this
+[repository](https://github.com/stackhpc/kubespray).
+
+After the notebook is deployed, a hyperlink to the notebook will be
+shown on the ansible logs as the final task.
+
+# Pre-requisite:
+
+Ensure that you cofigure your `~/.ssh.config` includes something similar to
+the following configuration where `ilab-gate` is your gateway node which
+is on the same subnet as your kubernetes cluster.
+
+```
```

Comment:
Would be nice to not make this alaska-specific. What are the key requirements for the environment that we need to convey here? The existence of a gateway host, which has access to the k8s API? Could this gateway host be localhost if the API is directly accessible?

Reply:
Yes, the gateway could be localhost, I can safely say after having played around with Ansible for a couple of more months.